lib/generators/rom/repository/templates/repository.rb.erb

```diff
@@ -0,0 +1,3 @@
+class <%= model_name %>Repository < ROM::Repository<%="[:#{relation}]"%>
+  commands :create, :update, :delete, mapper: :<%= mapper %>
```

Comment:
This should be `commands :create, update: :by_pk, delete: :by_pk, mapper: :<%= mapper %>`

Otherwise update/delete commands will be crashing